command/action.go

```diff
@@ -100,6 +93,8 @@ func (c *ActionCommand) AutocompleteArgs() complete.Predictor {
 
 func (c *ActionCommand) Name() string { return "action" }
 
+const defaultEscapeChar = "~"
```

Comment:
Usually we define constants at the top of the file, so above `type ActionCommand struct {` in this case.